components/Balance/Balance.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from "../Balance/Balance.module.css";
+
+export type BalanceProps = {
+  totalValue: string;
+  returnValue: string;
+};
+
+function Balance({ totalValue, returnValue }: BalanceProps) {
+  return (
+    <div className={styles.balanceContainer}>
+      <div className={styles.balanceHeader}>Balance</div>
+      <div className={styles.totalValue}>${totalValue}</div>
```

Comment:
If you like to format the number to add `+` for positive values, this is the right position.